scalastyle-config.xml

```diff
@@ -10,12 +10,6 @@
     <check level="warning" class="org.scalastyle.scalariform.SpacesAfterPlusChecker" enabled="true"/>
     <check level="warning" class="org.scalastyle.file.WhitespaceEndOfLineChecker" enabled="true"/>
     <check level="warning" class="org.scalastyle.scalariform.SpacesBeforePlusChecker" enabled="true"/>
-    <check level="warning" class="org.scalastyle.file.FileLineLengthChecker" enabled="true">
```

Comment:
I didn't think we were removing this.

Reply:
I think you can either remove it or set it to false. I've added it back in but set it to false